src/lib/pickles/common.ml

```diff
@@ -228,20 +228,44 @@ let ft_comm ~add:( + ) ~scale ~endoscale ~negate
       (snd (Dlog_plonk_types.Permuts_minus_1.add Nat.N1.n))
   in
   let f_comm =
+    (* the poseidon and generic gates are special cases as they use  coefficient commitments from the verifier index *)
```

Comment:
Nit: our usual comment style is
```ocaml
(* Comment begins with a capital and ends with a period. *)
(* Comments that span multiple lines have their closing parenthesis on a new
   line. The comment should have line breaks at column 80, as with normal code,
   and the text on subsequent lines should be aligned with the start of the
   comment text in the initial line.
*)


(* Naturally, exceptions apply. For example, you might want to end with some
   other piece of punctuation:
*)
(* Especially when it is more natural to start describing something ... *)
(* ... and then continue after some interleaved code, so that the most relevant
   details stay proximate to the corresponding code.
*)
```